corgi/core/models.py

```diff
@@ -46,6 +51,7 @@ class MPTTMeta:
     #
     # - Each method assumes that an object model will always link to a single ProductNode (thus
```

Comment:
those methods probably need to be fixed rather then adulterate the model.

Reply:
I wasn't proposing any model changes, just noting that our tree will no longer look like a tree and trying to find e.g. all linked ProductStreams may return odd results.